app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -15,35 +18,86 @@ $image-overlay-form-width: 350px;
   max-width: 100%;
 }
 
+.alchemy-image-overlay-dialog {
+
+  &.hide-form {
+
+    .picture-details-overlay {
+      right: -$image-overlay-form-width;
+    }
+
+    .zoomed-picture-background {
+      padding-right: 2*$default-padding;
```

Comment:
`2*$default-padding` should be written with a single space on each side of the operator: `2 * $default-padding`